api/v1beta1/openstackdataplaneservice_types.go

```diff
@@ -45,15 +45,6 @@ type KubeService struct {
 
 // OpenStackDataPlaneServiceSpec defines the desired state of OpenStackDataPlaneService
 type OpenStackDataPlaneServiceSpec struct {
-	// Label to use for service.
-	// Must follow DNS952 subdomain conventions.
-	// Since we are using it to generate the pod name,
-	// we need to keep it short.
-	// +kubebuilder:validation:Optional
-	// +kubebuilder:validation:Pattern="[a-z]([-a-z0-9]*[a-z0-9])"
-	// +kubebuilder:validation:MaxLength=53
```

Comment:
we can move it to Name

Reply:
can we? Name is part of metav1.ObjectMeta, and already has its own Kubernetes validations. https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names

Reply:
my bad, the Name I saw in `api/v1beta1/openstackdataplaneservice_types.go` was from KubeService